src/modules/commander/commander_tests/state_machine_helper_test.cpp

```diff
@@ -351,31 +353,31 @@ bool StateMachineHelperTest::mainStateTransitionTest()
 
 		{
 			"transition: MANUAL to ACRO - rotary",
-			MTT_ROTARY_WING,
+			MTT_ROTARY_WING | MTT_ANGULAR_VELOCITY_VALID,
 			commander_state_s::MAIN_STATE_MANUAL, commander_state_s::MAIN_STATE_ACRO, TRANSITION_CHANGED
 		},
 
 		{
 			"transition: MANUAL to ACRO - not rotary",
-			MTT_ALL_NOT_VALID,
+			MTT_ALL_NOT_VALID | MTT_ATTITUDE_VALID | MTT_ANGULAR_VELOCITY_VALID,
```

Comment:
Why do we need attitude for fixedwing acro?

Reply:
Because `fw_att_control` only runs when there's attitude.

Yes this should be fixed (separately).